### ISSUE TITLE
Additions to Application Area workflow

### ISSUE THREAD

**Issue by rlwilliamss** [archesproject/arches-her#679]
We would like to make the following additions to the Application Area workflows: 

_- (priority 1 enhancement) - _Add Administrative Area to the Application Area workflow Address section.__ No longer needed.

_- (priority 1 enhancement) - Location Description to the Application Area workflow Address section._ - Keep in Area Description tab as it is now.

- (priority 2 enhancement) - In the Application Area workflow - can the ‘Show application area name’ be renamed ‘Edit Application Area Name’.

**Comment by chiatt** [comment]
@rlwilliamss Should the **Administrative Area** fields go above or below the address fields? 
Also, the Location Description and type are in the **Area Description** tab. I just want to confirm that we should delete that tab.

**Comment by rlwilliamss** [comment]
Hi @chiatt, I'm just checking to confirm we would still like to go ahead with these enhancements and then I'll get back to you.

**Comment by chiatt** [comment]
Sounds good, thanks @rlwilliamss

**Comment by rlwilliamss** [comment]
@chiatt could the Administrative Area fields go below the address fields, and can we just keep the Location Description and type in the Area Description tab as it is currently, thank you

**Comment by rlwilliamss** [comment]
Hi @chiatt, could we actually just leave the address fields as they are currently, and not add the Admin Area field, thank you.

**Comment by StucakeGLHER** [comment]
@chiatt @rlwilliamss HI,  Yes, I aggree, lets keep the address fiels as its for the moment.  There have been some questions about this from intial Adviser and Business Officer testing and training, but I would rather keep as is until they and my team have had a chance to really use Arches before we start changing things.

**Comment by chiatt** [comment]
I'm going to close this for now, but please reopen if any changes are needed.
